### PR TITLE
Smoke tests: fix live check url depending on deployment context

### PR DESCRIPTION
**Pull request by csovant**
Jira issue? none
Release 6.5.0 backport required? no
6.5.0 doc changes needed? no
7.0.0 doc changes needed? no
Required README updates made? no

Manage the live check url depending on deployment context, use `https` in minikube context and `http` in GKE one

### CODE REVIEW

File: cicd/forgeops-tests/config/ProductConfig.py [hunk 1]
```diff
@@ -138,17 +150,6 @@ def start_ds_port_forward(self, instance_nb=0):
             raise Exception('Port-forward for pod %s on port %s not ready after %ss' %
                             (ds_pod_name, ds_local_port, duration))
 
-    @staticmethod
```
Comment (markg58) [inline cicd/forgeops-tests/config/ProductConfig.py]: Curious, why did you remove this and opt for the utils.cmd()?

Comment (markg58) [reply cicd/forgeops-tests/config/ProductConfig.py]: ah, nvm.  I see you moved it to a lib

Comment (csovant) [reply cicd/forgeops-tests/config/ProductConfig.py]: Yep I had to create a new method for foreground processes
